arch/arm64/src/common/arm64_syscall.c

```diff
@@ -217,10 +217,13 @@ uint64_t *arm64_syscall(uint64_t *regs)
 
       case SYS_switch_context:
         {
+          struct tcb_s *rtcb = (struct tcb_s *)regs[REG_X1];
+
           DEBUGASSERT(regs[REG_X1] != 0 && regs[REG_X2] != 0);
-          *(uint64_t **)regs[REG_X1] = regs;
+          tcb = (struct tcb_s *)regs[REG_X2];
```

Comment:
Please move to line 221 and add line change after

```
struct tcb_s *rtcb = (struct tcb_s *)regs[REG_X1];
tcb = (struct tcb_s *)regs[REG_X2];

DEBUGASSERT(regs[REG_X1] != 0 && regs[REG_X2] != 0);
```

Reply:
done